pkgs/development/haskell-modules/configuration-ghcjs-9.x.nix

```diff
@@ -24,6 +24,10 @@ with haskellLib;
   ghcjs-dom-javascript = addBuildDepend self.ghcjs-base super.ghcjs-dom-javascript;
   jsaddle = addBuildDepend self.ghcjs-base super.jsaddle;
   jsaddle-dom = addBuildDepend self.ghcjs-base super.jsaddle-dom;
+  jsaddle-warp = overrideCabal (drv: {
+    libraryHaskellDepends = [ ];
+    testHaskellDepends = [ ];
+  }) super.jsaddle-warp;
```

Comment:
Do you mind adding a comment here? I have no clue what this is achieving.

Reply:
Same as almost everything in this file: https://github.com/NixOS/nixpkgs/blob/465ba967e2c5c17724d54450cc178bbd52b96165/pkgs/development/haskell-modules/configuration-ghcjs-9.x.nix#L9

The only exception is the `disableParallelBuilding` overrides, which are currently needed for 9.8 and 9.10. 

Maybe the two override groups can be split more clearly?